src/test/kotlin/org/jetbrains/gradle/ext/BuildIdeArtifactTest.kt

```diff
@@ -244,11 +250,19 @@ class BuildIdeArtifactTest {
   }
 
   fun createTestArchive(sourceDir: File, destinationFile: File) {
-    myProject.tasks.create("testArchive", Zip::class.java) {
-      it.from(sourceDir)
-      it.destinationDir = destinationFile.parentFile
-      it.archiveName = destinationFile.name
-    }.execute()
+    var out = ZipOutputStream(BufferedOutputStream(FileOutputStream(destinationFile)))
```

Comment:
I assume Kotlin has a try-with-resources syntax which could be used here?

Reply:
there is `use`. Replaced it to use this syntax